index.html

```diff
@@ -157,12 +158,28 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
           <li>CSS</li>
           <li>Ruby</li>
         </ul>
+        <button type="button" class="see-project-card" id="button-5">
+          See Project
+        </button>
+      </div>
+      <div class="card-content" id="card-content-6">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>CSS</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button" class="see-project-card" id="button-6">
+          See Project
+        </button>
       </div>
-      <button type="button" class="see-project-card">
-        See Project
-      </button>
     </div>
   </section>
+
   <section class="about-me" id="about-me">
     <div class="resume">
       <h1>About me</h1>
```

Comment:
- [ ] Kindly add left margin/padding to try make it look like in the Figma design.
#### Design
![about-section-left-figma](https://user-images.githubusercontent.com/44057863/167276244-76170f13-c3da-4dab-8230-1d02df30b005.png)
#### Impl
![about-section-left-impl](https://user-images.githubusercontent.com/44057863/167276252-e8202e24-2953-42c1-bd72-ddc840599d40.png)

Reply:
Thanks for the review.